.github/workflows/zowe-cli-plugin.yml

```diff
@@ -59,7 +59,7 @@ jobs:
 
     - name: Archive Results
       if: ${{ always() && steps.build.outcome == 'success' }}
-      uses: actions/upload-artifact@v2
+      uses: actions/upload-artifact@v3
       with:
         name: ${{ matrix.os }}-${{ matrix.node-version }}-results
         path: |
```

Comment:
could you also update the codecov-action to v3 here?

Reply:
This still needs to be updated